pyswarms/utils/environments/plot_environment.py

```diff
@@ -104,7 +104,7 @@ def assertions(self):
                 "Missing methods in optimizer, check " "pyswarms.base module"
             )
 
-    def __init__(self, optimizer, objective_func, iters):
+    def __init__(self, optimizer, objective_func, iters, obj_args=(), obj_kwargs={}):
         """Runs the optimizer against an objective function for a number
```

Comment:
No need to touch this :+1: This will be Deprecated soon. Simply revert this to the original to avoid merge conflicts down the road.